lib/cadet_web/views/view_helpers.ex

```diff
@@ -3,8 +3,36 @@ defmodule CadetWeb.ViewHelpers do
   Helper functions shared throughout views
   """
 
-  def format_datetime(datetime) do
-    Timex.format!(DateTime.truncate(datetime, :millisecond), "{ISO:Extended}")
+  alias Timex.Timezone
+
+  def grader_builder(nil), do: nil
+
+  def grader_builder(_) do
+    fn %{grader: grader} ->
+      transform_map_for_view(grader, [:name, :id])
+    end
+  end
+
+  def graded_at_builder(nil), do: nil
+
+  def graded_at_builder(_) do
+    fn %{updated_at: updated_at} ->
+      format_datetime(updated_at)
+    end
+  end
+
+  def format_datetime(datetime = %DateTime{}) do
+    datetime
+    |> DateTime.truncate(:millisecond)
+    |> Timex.format!("{ISO:Extended}")
+  end
+
+  def format_datetime(datetime = %NaiveDateTime{}) do
+    timezone = Timezone.get("Asia/Singapore", Timex.now())
```

Comment:
Shit. I don't think this is correct. I think naive date time stores in UTC. Help me double check